api/swagger-spec/oapi-v1.json

```diff
@@ -13906,6 +13906,10 @@
       "type": "string",
       "description": "type of source control management system"
      },
+     "dockerfile": {
+      "type": "string",
+      "description": "the contents of a Dockerfile to build; FROM and ENV may be overriden if you have specified 'from' and 'env' on the build strategy"
```

Comment:
ENV won't be overridden (btw. you have typo there), but it will be just appended to Dockerfile. If you define ENV in your dockerfile, it will be available as we append our ENV right after FROM @rhcarvalho FYI.